public/app/plugins/datasource/cloudwatch/components/SQLBuilderEditor/SQLFilter.tsx

```diff
@@ -104,12 +104,12 @@ const FilterItem: React.FC<FilterItemProps> = (props) => {
   const dimensionKeys = useDimensionKeys(datasource, { region: query.region, namespace, metricName });
 
   const loadDimensionValues = async () => {
-    if (!filter.property?.name) {
+    if (!filter.property?.name || !namespace) {
```

Comment:
This is why namespace is no longer optional?

Reply:
Yes